rdf-abac-core/src/main/java/io/telicent/jena/abac/labels/LabelsStoreRocksDB.java

```diff
@@ -536,7 +589,19 @@ private void addRule___(final List<String> labelsList) {
         labelMode.writeUsingMode(txRocksDB, cfh___, KEY_cfh___, labels.flip());
     }
 
-    // Information gathering counts of number of number each pattern type is used.
+    /**
+     * Remove any labels for a specific triple.
+     * This does not affect any patterns.
+     */
+    @Override
+    public void remove(Triple triple) {
+        tripleLabelCache.remove(triple);
+        throw new UnsupportedOperationException("LabelsStore.remove not supported by LabelsStoreRockDB");
```

Comment:
Would we ever need to support this?  Or would we mostly just overwrite an existing label with a new label?

Reply:
As currently used by Telicent, labels are not deleted except by the act of replacing a triple/label mapping.

But in the future that might change so the interface has an operation. For example, a store used with a fixed default label  covering the majority of data and a this is a `LabelsStore` of exceptions to the default. Then label change can involve removing an entry.

This is here to put in the fact the cache would need updating if the full operation is implemented.